mlflow/deployments/server/app.py

```diff
@@ -330,6 +330,65 @@ async def get_limits(endpoint: str) -> LimitsConfig:
     async def set_limits(payload: SetLimitsModel) -> LimitsConfig:
         raise HTTPException(status_code=501, detail="The set_limits API is not available yet.")
 
+    def _look_up_route(name: str) -> Optional[Route]:
+        if r := app.dynamic_routes.get(name):
+            return r
+
+        raise HTTPException(
+            status_code=404,
+            detail=f"Route {name} not found in the configuration.",
+        )
+
+    # OpenAI compatible endpoints
+    @app.post("/v1/chat/completions")
+    async def chat_handler(request: Request, payload: chat.RequestPayload) -> chat.ResponsePayload:
```

Comment:
Can we call these `openai_chat_handler`, `openai_completions_handler`, etc?

Reply:
sg